### PR TITLE
Added changes in UI of admin_add_login page

### LINKED ISSUE #3746: Admin interface registration form should list username limitations
## Description

Username field should have some smol help text indicating whether a username can have spaces in it, is case-sensitive, etc.

## User Research Evidence

During SecureDrop trainings, one of the questions asked by admins and journalists creating accounts on the SecureDrop admin interface is whether they can have spaces in their username or if the username is case-sensitive. The form should indicate these things.

### PR DESCRIPTION
**Pull request by nightwarrior-xxx**
## Status

Ready for review 

## Description of Changes

Fixes #3746 

Changes proposed in this pull request:
* Changed the UI of the admin_add_user page

## Testing

How should the reviewer test this PR?
Write out any special testing steps here.

Have a look at the admin_add_user page

### CODE REVIEW

**Comment by nightwarrior-xxx** [comment]
Have a look a screen shot
screen_shot_of_changes made

**Comment by nightwarrior-xxx** [comment]
@heartsucker Please review!
Regards,
@nightwarrior-xxx

**Comment by heartsucker** [comment]
Hey @nightwarrior-xxx. There's still two diffs that are just whitespace. Also, could you squash your commits so this PR is just a single commit? It makes the history a little cleaner.

File: securedrop/journalist_templates/admin_add_user.html [hunk 1]
```diff
@@ -33,9 +45,9 @@
...
     <br>
     {% for error in form.is_hotp.errors + form.otp_secret.errors %}
-      <span class="form-validation-error">{{ error }}</span><br>
+    <span class="form-validation-error">{{ error }}</span><br>
     {% endfor %}
   </p>
   <button type="submit" class="sd-button"><i class="fa fa-plus"></i>{{ gettext('ADD USER') }}</button>
 </form>
-{% endblock %}
+{% endblock %}
```
Comment (SaptakS) [inline securedrop/journalist_templates/admin_add_user.html]: I think you should keep the existing newline in the file.

**Comment by nightwarrior-xxx** [comment]
Hey @heartsucker,I just did indentation,which resulted in the removing of 2 white space you are talking about .
Regards,
@nightwarrior-xxx

**Comment by heartsucker** [comment]
Heya. Some more notes.

screenshot_2018-10-10_20-03-23

I still think the help text is a little too small. My laptop is kinda shitty, so maybe it's just that the text doesn't render great, but i think this could be a problem for other users.

It would be nice to make the grey background a little bigger (using padding/margin) so that it's not hugging the text so tight.

**Comment by nightwarrior-xxx** [comment]
Hey @heartsucker ,Cool I will do that.But what about those 2 spacing problems ,I just did indentation ,still do I change it as you said?
Regards,
@nightwarrior-xxx

**Comment by heartsucker** [comment]
I think we need to call in the UX experts to make an assessment here. I'm mostly backend, so we'll need more input from people who can make a call on this.

**Comment by ninavizz** [comment]
Ok! @SaptakS will you be in the UX call, in the AM PST? I think I get what's being asked-for here, but it's always nice to chat needs through. Regardless, can follow-up tomorrow late afternoon PST with whatever may be needed. Cheerio!

**Comment by nightwarrior-xxx** [comment]
@heartsucker Should I close this PR and made a new PR in order to not to make history.Moreover I am facing issue in pulling also.

File: securedrop/sass/journalist.sass [hunk 2]
```diff
@@ -101,3 +101,15 @@ button.small-danger, a.btn.small-danger, .btn.small-danger
 
 .journalist-reply__input
   max-width: 700px
+
+.journalist-username__notes
+  font: 12px Sans-Serif
+  font-weight: normal
+  letter-spacing: 0.05px
+
+.code_modified
```
Comment (SaptakS) [inline securedrop/sass/journalist.sass]: Firstly, you need to put this code [here](https://github.com/freedomofpress/securedrop/blob/develop/securedrop/sass/modules/_code.sass). Secondly, what is the benefit of removing the `padding` and the `border-bottom`? @ninaviz what do you think about this design vs the one we have for `.code` ?

Comment (nightwarrior-xxx) [reply securedrop/sass/journalist.sass]: @SaptakS I removed padding for highlighted password part as it was taking more space on the upper side but later after @heartsucker said that font-size is too small so I increased the font size and added padding with suitable pixels.I will push PR after modifying it again.

**Comment by heartsucker** [comment]
The current UI looks like this which I think is nearly there.

screenshot_2018-10-12_14-51-16

For the UX team, my one open question is: what should we do about multi-line passphrase? Because right now someone might forget to copy the last word of the passphrase and then not be able to log in.

**Comment by nightwarrior-xxx** [comment]
@heartsucker I made a new PR because I was unable to fetch changes,which I tried hard to fetch but failed.New issue no. is #3873 .I will not repeat this mistake again.

Moreover,we can increase the border size of the form ,or reduce the length of password by one word if we want the password to be in one line
Regards,
@nightwarrior-xxx

**Comment by eloquence** [comment]
Closing as duplicate of #3873.

**Pull request closed without merging**